Admin.js

```diff
@@ -114,561 +141,775 @@ var Admin = {
-        var ereg = /width:(auto|(([-+]?([0-9]*\.)?[0-9]+)(px|em|ex|%|in|cm|mm|pt|pc)))/i;
+    "get_select2_width" (element) {
+
+        const ereg = /width:(auto|(([-+]?([0-9]*\.)?[0-9]+)(px|em|ex|%|in|cm|mm|pt|pc)))/i;
 
-        // this code is an adaptation of select2 code (initContainerWidth function)
-        var style = element.attr('style');
-        //console.log("main style", style);
+        // This code is an adaptation of select2 code (initContainerWidth function)
+        let style = element.attr("style");
+        // Console.log("main style", style);
 
         if (style !== undefined) {
-            var attrs = style.split(';');
 
-            for (var i = 0, l = attrs.length; i < l; i = i + 1) {
-                var matches = attrs[i].replace(/\s/g, '').match(ereg);
-                if (matches !== null && matches.length >= 1)
+            const attrs = style.split(";");
+
+            for (let i = 0, l = attrs.length; i < l; i += 1) {
+
+                const matches = attrs[i].replace(/\s/g, "").match(ereg);
+                if (matches !== null && matches.length >= 1) {
+
                     return matches[1];
+
+                }
+
             }
+
         }
 
-        style = element.css('width');
+        style = element.css("width");
         if (style.indexOf("%") > 0) {
+
             return style;
+
         }
 
-        return '100%';
+        return "100%";
+
     },
 
-    setup_sortable_select2: function(subject, data) {
-        var transformedData = [];
-        for (var i = 0 ; i < data.length ; i++) {
-            transformedData[i] = {id: data[i].data, text: data[i].label};
+    "setup_sortable_select2" (subject, data) {
+
+        const transformedData = [];
+        for (let i = 0; i < data.length; i++) {
+
+            transformedData[i] = {
+                "id": data[i].data,
+                "text": data[i].label
+            };
+
         }
 
         subject.select2({
-            width: function(){
-                // Select2 v3 and v4 BC. If window.Select2 is defined, then the v3 is installed.
-                // NEXT_MAJOR: Remove Select2 v3 support.
+            "width" () {
+
+                /*
+                 * Select2 v3 and v4 BC. If window.Select2 is defined, then the v3 is installed.
+                 * NEXT_MAJOR: Remove Select2 v3 support.
+                 */
                 return Admin.get_select2_width(window.Select2 ? this.element : subject);
-            },
-            dropdownAutoWidth: true,
-            data: transformedData,
-            multiple: true
-        });
 
-        subject.select2("container").find("ul.select2-choices").sortable({
-            containment: 'parent',
-            start: function () {
-                subject.select2("onSortStart");
             },
-            update: function () {
-                subject.select2("onSortEnd");
-            }
+            "dropdownAutoWidth": true,
+            "data": transformedData,
```

Comment:
- eslint: Expected object keys to be in ascending order. 'data' should be before 'dropdownAutoWidth'.